driver-core/src/main/com/mongodb/connection/InternalStreamConnection.java

```diff
@@ -656,82 +640,14 @@ public void onResult(final ByteBuf result, final Throwable t) {
         }
     }
 
-    private class CommandEventSender {
```

Comment:
Moved to the top level to make it easier to unit test.

Reply:
Ack